packages/ckeditor5-core/src/editor/editorconfig.ts

```diff
@@ -821,6 +821,44 @@ export interface EditorConfig {
 	 * Translations to be used in the editor.
 	 */
 	translations?: ArrayOrItem<Translations>;
+
+	/**
+	 * Callback used to sanitize the HTML provided by the user when generating previews of it in the editor.
+	 *
+	 * We strongly recommend overwriting the default function to avoid XSS vulnerabilities.
+	 *
+	 * Read more about the security aspect of this feature in the {@glink features/html/html-embed#security "Security"} section of
+	 * the {@glink features/html/html-embed HTML embed} feature guide.
+	 *
+	 * The function receives the input HTML (as a string), and should return an object
+	 * that matches the {@link module:html-embed/htmlembedconfig~HtmlEmbedSanitizeOutput} interface.
+	 *
+	 * ```ts
+	 * ClassicEditor
+	 *   .create( editorElement, {
+	 *     htmlEmbed: {
+	 *       showPreviews: true,
+	 *       sanitizeHtml( inputHtml ) {
+	 *         // Strip unsafe elements and attributes, e.g.:
+	 *         // the `<script>` elements and `on*` attributes.
+	 *         const outputHtml = sanitize( inputHtml );
+	 *
+	 *         return {
+	 *           html: outputHtml,
+	 *           // true or false depending on whether the sanitizer stripped anything.
+	 *           hasChanged: ...
+	 *         };
+	 *       },
+	 *     }
+	 *   } )
+	 *   .then( ... )
+	 *   .catch( ... );
+	 * ```
+	 *
+	 * **Note:** The function is used only when the feature
```

Comment:
Rewrite this part. It should say something like:

This function is used by following features:

*   {@link HTML embed} (when {@link showPreviews} flag is set)
*   {@link Merge fields} (when {@link previewHtmlValues} flag is set).